CsprSdkStandardTestsNet/Test/Steps/Deploys.cs

```diff
@@ -285,14 +290,9 @@ public class Deploys {
         Assert.That(namedArg, Is.Not.Null);
         
         var userKeyAsset = AssetUtils.GetUserKeyAsset(1, user, StepConstants.PUBLIC_KEY_PEM);
-
         var publicKey = PublicKey.FromPem(userKeyAsset);
 
-        //Have to convert the PublicKey to an Account Hash
-        var accountHash = publicKey.GetAccountHash();
-        
-        // This only works if we compare the Account Hash of the user's PublicKey
-        Assert.That(namedArg.Value.Parsed.ToString()?.ToUpper(), Is.EqualTo(accountHash[13..].ToUpper()));
+        Assert.That(namedArg.Value.Parsed.ToString()?.ToUpper(), Is.EqualTo(publicKey.ToString().ToUpper()));
```

Comment:
Use .IgnoreCase modifier:
`  Assert.That(namedArg.Value.Parsed.ToString(), Is.EqualTo(publicKey.ToString()).IgnoreCase);`

See NUnit docs https://docs.nunit.org/2.5/equalConstraint.html

Reply:
yeah I missed that
I'll create a new issue to replace all these in the project